Changes

```diff
@@ -162,6 +162,9 @@ Next version (4.05.0):
 - PR#5163: ocamlobjinfo, dump globals defined by bytecode executables
   (Stéphane Glondu)
 
+- PR#7060: toplevel, print exceptions in installed custom printers
```

Comment:
Please include GPR number as well;

`PR#7060, GPR#1035: print name of the exception raised in custom printers.`

It should also be in section `### Toplevel and debugger:` (which will be a new section for this release)